src/hidimstat/ensemble_clustered_inference.py

```diff
@@ -144,6 +150,13 @@ def ensemble_clustered_inference(
            Spatially relaxed inference on high-dimensional linear models.
            arXiv preprint arXiv:2106.02590.
     """
+
+    if memory is not None and not isinstance(memory, str):
```

Comment:
same as above, shouldn't we prefer sklearn's `check_memory`

Reply:
No, because the check of memory creates an object memory from joblib. This is not what we want to do here.

Reply:
There was missing the case where the memory are an instance of jolib.memory

Reply:
good point.